README.md

```diff
@@ -89,6 +89,8 @@ being too big for intelliSense to work.*
 |4.1|[https://pypi.org/project/fake-bpy-module-4.1/](https://pypi.org/project/fake-bpy-module-4.1/)|
 |4.2|[https://pypi.org/project/fake-bpy-module-4.2/](https://pypi.org/project/fake-bpy-module-4.2/)|
 |4.3|[https://pypi.org/project/fake-bpy-module-4.3/](https://pypi.org/project/fake-bpy-module-4.3/)|
+|4.4|[https://pypi.org/project/fake-bpy-module-4.4/](https://pypi.org/project/fake-bpy-module-4.4/)|
```

Comment:
New blender version support should be done in the separated PR.
Please focus on the current version for this PR.

Reply:
Np, removed new Blender versions from the PR.